### PR TITLE
Enum bscore

### PR DESCRIPTION
**Pull request by KirosHailay**

### CODE REVIEW

File: tests/moses/scoring/enum_bscoreUTest.cxxtest [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		ValuePtr f3_proto_atom(new LinkValue(_f3_values));
+		f3->setValue(moses::compressed_value_key, f3_proto_atom);
+
+		HandleSeq handleSeq = {f1, f2, f3};
+		Handle program = createLink(handleSeq, COND_LINK);
+
+		behavioral_score bs = et_bscore(program);
+		// note that the compressed table might get reordered in this case the
+		// fourth row will come at the first, first row will come at the second
+		// , third row will come at  the Since the population of the input data
```
Comment (ngeiswei) [inline tests/moses/scoring/enum_bscoreUTest.cxxtest]: 1. `,` shouldn't be at the beginning of a line.
2. "come at   the", looks like some word is missing here.

File: tests/moses/scoring/enum_bscoreUTest.cxxtest [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		// to atomspace is called after the reorder this should have no problem.
+		behavioral_score expected_bs = {-7, -4, -1, -8};
+
+		TS_ASSERT_EQUALS(bs, expected_bs);
+	}
+};
```
Comment (ngeiswei) [inline tests/moses/scoring/enum_bscoreUTest.cxxtest]: Should end with a newline.

**Comment by kasimebrahim** [comment]
Can you try it with this test case.
```
out,f1,f2,f3 
high,1,1,0
low,1,1,0
mid,0,0,0
low,0,0,0
```

**Comment by ngeiswei** [comment]
Looks good to me, thanks.

**Pull request merged**